network-api/networkapi/wagtailpages/pagemodels/publications/publication.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+from wagtail.core.models import Page
+
+from ..mixin.foundation_metadata import FoundationMetadataPageMixin
+
+
+class PublicationPage(FoundationMetadataPageMixin, Page):
+    """
+    This is the root page of a publication. From here the user can browse to the various sections (called chapters).
+    It will have information on the publication, its authors, and metadata from it's children
+    """
+
+    # TODO: once we make chapter pages, add that as a subpage type
+    subpage_types = []
```

Comment:
are publication pages not also their own child? (I don't actually know, but I'd assume they are)

Reply:
There may not actually be many if any differences between chapter pages and publication pages(though there are some differences on the frontend). I think thats what you're getting at?

Reply:
No, just genuine curiosity: I know that campaigns can be children of campaigns, so I was wondering if publications were supposed to be children of publications, but whether or not they do won't hold up this PR